src/apispec/ext/marshmallow/field_converter.py

```diff
@@ -402,14 +403,33 @@ def nested2properties(self, field, ret):
         :param Field field: A marshmallow field.
         :rtype: dict
         """
-        if isinstance(field, marshmallow.fields.Nested):
+        # Pluck is a subclass of Nested but is in essence a single field; it
+        # is treated separately by pluck2properties.
```

Comment:
You have two options:
- use strict type checks (`type(field) is marshmallow.fields.Nested`). This would break any field inheriting from Nested without changing semantics the way `Pluck` does.
- factor out the common functionality that Pluck needs to reuse from Nested into a common utility class.

The latter would be my choice. It may, however, break 3rd-party code relying on `isinstance(pluck_field, Nested)` to be true.

Reply:
Yes, exactly.

We shall investigate not having `Pluck` inherit `Nested`, but that would probably be a marshmallow 4 change. Meanwhile, we can live with this test in apispec.